openshift/koku-template.yaml

```diff
@@ -53,12 +53,18 @@ objects:
       template.openshift.io/expose-password: "{.data['database-password']}"
       template.openshift.io/expose-username: "{.data['database-user']}"
       template.openshift.io/expose-django_secret_key: "{.data['django-secret-key']}"
+      template.openshift.io/expose-service_admin_username: "{.data['service-admin-user']}"
+      template.openshift.io/expose-service_admin_email: "{.data['service-admin-email']}"
+      template.openshift.io/expose-service_admin_password: "{.data['service-admin-password']}"
     name: ${DATABASE_SERVICE_NAME}-secret
   stringData:
     database-password: ${DATABASE_PASSWORD}
     database-user: ${DATABASE_USER}
     django-secret-key: ${DJANGO_SECRET_KEY}
     database-admin-password: ${DATABASE_ADMIN_PASSWORD}
+    service-admin-user: ${SERVICE_ADMIN_USER}
+    service-admin-email: ${SERVICE_ADMIN_EMAIL}
+    service-admin-password: ${SERVICE_ADMIN_PASSWORD}
```

Comment:
Putting the django secrets in with the pgsql secrets seems like it's expanding the scope of the object. We should either rename it to something other than "${DATABASE_SERVICE_NAME}-secret" or we should create a new secret store for the app-specific secrets.

Reply:
👍 I'll update this. Was a part of the template I didn't quite follow.